PyAlbany/src/Albany_Pybind11_Tpetra.hpp

```diff
@@ -0,0 +1,55 @@
+//*****************************************************************//
+//    Albany 3.0:  Copyright 2016 Sandia Corporation               //
+//    This Software is released under the BSD license detailed     //
+//    in the file "license.txt" in the top-level Albany directory  //
+//*****************************************************************//
+
+#ifndef PYALBANY_TPETRA_H
+#define PYALBANY_TPETRA_H
+
+#include "Albany_TpetraTypes.hpp"
+#include "Albany_Pybind11_Comm.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/numpy.h>
+#include <pybind11/stl.h>
+
+using RCP_PyMap = Teuchos::RCP<Tpetra_Map>;
+using RCP_ConstPyMap = Teuchos::RCP<const Tpetra_Map>;
+using RCP_PyVector = Teuchos::RCP<Tpetra_Vector>;
+using RCP_PyMultiVector = Teuchos::RCP<Tpetra_MultiVector>;
+
+
```

Comment:
can you please add descriptions for these functions pointing out any nonintuitive behavior?